listener/Cargo.toml

```diff
@@ -16,6 +16,8 @@ graph-store-postgres = { git = "https://github.com/graphprotocol/graph-node", ta
 graph-server-http = { git = "https://github.com/graphprotocol/graph-node", tag = "v0.13.2" }
 graph-server-websocket = { git = "https://github.com/graphprotocol/graph-node", tag = "v0.13.2" }
 
+dfusion_core = { path = "../dfusion_rust_core" }
```

Comment:
I feel like these kind of imports would have already been included? Is this part of a different PR?

Reply:
This is the first PR where the listener actually uses the common library (before I only created it and made the driver use it).